src/main/java/com/gascalc/GasCalcController.java

```diff
@@ -20,9 +30,13 @@ public String searchForm(Model model) {
 		return "search";
 	}
 	
-	
+	/**
+	 * 
+	 * @param model model to represent the returned GasCalcModel
+	 * @return search results for the car search form
+	 */
```

Comment:
As stated above, including javadocs can really pay dividends in the future once your project becomes more complicated. Right now, all of your code is very readable, however it is a good idea to build a habit of documenting things appropriately.

Reply:
I completely agree with this and added this to my PR as well addressing it in other places.